src/tuner/nccl_ofi_tuner.c

```diff
@@ -1,19 +1,20 @@
 /*
  * Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All rights reserved.
  */
-#include "config.h"
-
 #include <pthread.h>
 #include <stdio.h>
 #include <stdlib.h>
-#include <float.h>
-#include <math.h>
 
-#include "nccl-headers/nvidia/tuner.h"
+#include "config.h"
+
 #include "nccl_ofi_log.h"
 #include "nccl_ofi_math.h"
-#include "nccl_ofi_tuner.h"
 #include "nccl_ofi_pthread.h"
+#include "nccl_ofi_tuner.h"
+
+#include "nccl-headers/nvidia/tuner.h"
+#include <float.h>
```

Comment:
my preference (although unlike config.h being first, not going to block on it) is that the order be config.h, then system headers, then our headers.

Reply:
This is objectively wrong, thanks. My autoformatter did this.

Reply:
Un-resolving the conversation as this was not resolved :)

Reply:
Saw this in my email and kicked myself, but I think we're looking at an outdated diff -- I didn't have to make any changes here specifically. Double checked all files and I think it's right now.